README.md

```diff
@@ -1,46 +1,36 @@
-# Getting Started with Create React App
+# Portfolio Web Page - Beata Suścicka i Natalia Mateusiak
```

Comment:
Except for some small linguistic mistakes it's written well - good README is makes a good expression on the project reviewer. Good job with that

Reply:
I agree, the readme is great 👍🏻